initial-promises/node-server/common/1.0/site.cf

```diff
@@ -83,6 +83,9 @@ bundle common g
                                               "curl/libcurl.dll",
                                               "curl/libssl32.dll",
                                               "curl/libeay32.dll",
+                                              "openssl/openssl.exe",
+                                              "openssl/libeay32.dll",
```

Comment:
Do we really need to duplicate the libraries ? They are already present for curl.